Textile/Classes/Public/TextileApi.h

```diff
@@ -206,6 +206,19 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (Summary *)summary:(NSError **)error;
 
+/**
+ * Start the Textile node
+ * @param error A reference to an error pointer that will be set in the case of an error
+ * @return A boolean indicating if the node was started successfully
+ */
+- (BOOL)start:(NSError* _Nullable* _Nullable)error;
```

Comment:
Added `start` and `stopWithCompletion` to the public API. This means that, technically, someone can choose to start/stop the node directly. We will want this functionality to do something like implement a share extension in and iOS app.